### ISSUE TITLE
Module not found: Error: Cannot resolve module 'module'

### ISSUE THREAD

**Issue by johnpaularthur** [alm-tools/alm#330]
Project seems to run fine, but not sure what impact this error could have down the line.
Mac 10.11
Node v7.1.0

```
ERROR in ./~/source-map-support/source-map-support.js
Module not found: Error: Cannot resolve module 'module' in /Users/jarthur/git/alm-orig/node_modules/source-map-support
 @ ./~/source-map-support/source-map-support.js 464:17-34
```

**Comment by basarat** [comment]
Waiting on https://github.com/evanw/node-source-map-support/issues/155 :rose:

**Comment by johnpaularthur** [comment]
Ah,  any way to force it back to 0.4.3 until resolution?  Not sure which module requires it, i'm guessing some babel package.

**Comment by basarat** [comment]
`ts-node`. But I haven't seen it cause an issue to torture ts-node to track their external dependency :rose:

**Comment by basarat** [comment]
Released with `2.10.0` :rose:
